pom.xml

```diff
@@ -262,7 +263,7 @@
                     </historyOutputFile>
                     <verbose>true</verbose>
                     <targetClasses>
-                        <param>edu.ucsb.cs156.*</param>
+                        <param>${targetClasses}</param>
                     </targetClasses>
                     <targetTests>
                         <param>edu.ucsb.cs156.*</param>
```

Comment:
This should be in a separate PR.  Don't mix concerns that have nothing to do with each other in the same PR.

See: https://ucsb-cs156.github.io/topics/git/git_feature_branch_workflow.html#more-detail-about-creating-feature-branches

Reply:
Fixed!